### PR TITLE
Issue/20 preview release prep

### LINKED ISSUE #20: Preview release preparation
1. Add readme (done, therefore not part of this PR)
2. Update various github related items (done, not part of this PR)
3. Add code of conduct
4. Set license
5. Update `.csproj` file

### PR DESCRIPTION
**Pull request by lf-jason-jia**
Fix #20

1. Add code of conduct
2. Set license
3. Update `.csproj` file

### CODE REVIEW

File: src/Laserfiche.Repository.Api.Client.csproj [hunk 1]
```diff
@@ -2,6 +2,17 @@
...
+    <Authors>Laserfiche</Authors>
+    <Description>.NET API that enables easy and secure access to Laserfiche Repository.</Description>
+    <Copyright>Copyright 2022 Laserfiche</Copyright>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/Laserfiche/lf-repository-api-client-dotnet</PackageProjectUrl>
+    <PackageIcon>logo.png</PackageIcon>
+    <PackageIconUrl />
+    <RepositoryUrl>https://github.com/Laserfiche/lf-repository-api-client-dotnet</RepositoryUrl>
+    <RepositoryType>GitHub</RepositoryType>
+    <PackageTags>Laserfiche Repository Data Document-Management Document-Store</PackageTags>
```
Comment (jason-chan-lf) [inline src/Laserfiche.Repository.Api.Client.csproj:15]: Are these package tags relevant to the repo client lib? 
Data Document-Management Document-Store

Comment (lf-jason-jia) [reply src/Laserfiche.Repository.Api.Client.csproj:15]: I guess so? I can remove them but it's similar how we have tags for oauth client lib.

Comment (lf-jason-jia) [reply src/Laserfiche.Repository.Api.Client.csproj:15]: Just to add reasoning for the previous comment: The repp client lib connects to Laserfiche Repository which does store data and document and to some degree it allows document management.

**Review by jason-chan-lf — APPROVED** [review]
